admin/modules/user/users.php

```diff
@@ -992,7 +992,14 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
+		if(my_substr($mybb->settings['useravatar'], 0, 7) === 'http://' || my_substr($mybb->settings['useravatar'], 0, 8) === 'https://')
```

Comment:
Copying doesn't explain why it's correct. But it'll be changed to the new URL function soon so doesn't matter.